k8s/ingress.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: mongo-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /
+spec:
+  rules:
+  - host: www.dogimages.com

```

Comment:
Using the same host for all three different services, even tho they are not logically related?

Reply:
Added a different host for mongo-express and mongodb.